src/Workspaces/Core/Portable/FindSymbols/FindLiterals/FindLiteralsSearchEngine.cs

```diff
@@ -115,8 +134,7 @@ private async Task ProcessDocumentAsync(Document document, CancellationToken can
 
     private async Task ProcessDocumentWorkerAsync(Document document, CancellationToken cancellationToken)
     {
-        var index = await SyntaxTreeIndex.GetIndexAsync(
-            document, cancellationToken).ConfigureAwait(false);
+        var index = await SyntaxTreeIndex.GetIndexAsync(document, _storage, cancellationToken).ConfigureAwait(false);
```

Comment:
this is the main thrust of the change.  instead of the index being responsible for grabbing the storage and reading/writing from it, it is passed the storage instance.